app/main.py

```diff
@@ -1,2 +1,151 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            keys: list[Hashable] | None = None,
+            values: list[Any] | None = None
+    ) -> None:
+        self.__capacity = 8
+        self.__size = 0
+        self.__threshold = int((2 / 3) * self.__capacity)
+        self.__table = [(None,)] * self.__capacity
```

Comment:
Hash table should be a list of nodes. It would make sense to implement a separate Node class as per task requirements. It will help to better structure your implementation

> Also, don't forget to store (key, hash, value) as a node in the hash table. A hash table may be stored as a simple list of nodes.